### PR TITLE
Create .prettierrc.js

### PR DESCRIPTION
**Pull request by kylemh**
See: https://github.com/react-hookz/eslint-config/pull/2 for context!

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
```
Comment (xobotyi) [inline .prettierrc.js:1]: What if:
```
const eslintConfig = require("@react-hookz/eslint-config/.prettierrc.js")

module.exports = {
  ...eslintConfig,
};
```

This way it inherits the config used in eslint and easily can be extended

Comment (kylemh) [reply .prettierrc.js:1]: Good idea! I can also put it into `package.json` like you said on Discord, but the config file seems to follow what's been done with other similar tools.

Let me know!

Comment (xobotyi) [reply .prettierrc.js:1]: i'd stay with separate file.

**Comment by kylemh** [comment]
Done. Obviously, we'll want to update the version of the eslint-config package and upgrade here before merging.

**Comment by kylemh** [comment]
Whoops. Deleted fork.

**Pull request closed without merging**